sunpy/tests/figure_tests_env_2.7.yml

```diff
@@ -5,59 +5,70 @@ channels:
 - astropy
 dependencies:
 - ca-certificates=2016.9.26=0
-- contextlib2=0.5.1=py27_0
-- cycler=0.10.0=py27_0
-- decorator=4.0.10=py27_0
-- enum34=1.1.6=py27_1
-- glymur=0.8.4=py27_0
-- hypothesis=3.4.2=py27_0
-- icu=56.1=2
-- jpeg=9b=0
-- libiconv=1.14=2
-- libtiff=4.0.6=4
-- libxml2=2.9.3=7
-- networkx=1.11=py27_0
-- openjpeg=2.1.0=5
-- openssl=1.0.2h=1
-- pandas=0.18.1=np111py27_0
-- pillow=3.2.0=py27_1
-- pip=8.1.2=py27_0
-- pixman=0.32.6=0
-- py=1.4.31=py27_0
-- pytest=2.9.2=py27_0
-- python-dateutil=2.5.3=py27_0
-- pytz=2016.4=py27_0
-- requests=2.10.0=py27_0
-- scikit-image=0.12.3=np111py27_0
-- setuptools=23.0.0=py27_0
-- six=1.10.0=py27_0
-- sqlite=3.13.0=1
-- suds-jurko=0.6=py27_1
-- tk=8.5.19=0
-- wheel=0.29.0=py27_0
-- xz=5.0.5=1
-- zlib=1.2.8=3
+- glymur=0.8.8=py27_0
+- hypothesis=3.6.1=py27_0
+- openjpeg=2.1.2=2
```

Comment:
yes we moved it in here because the env needs to be co-versioned with the hashes, because otherwise we can't maintain a version for 0.7 etc.

I have never really liked the other repo, especially with @ayshih recent changes generating the old figures is as simple as `git checkout; python setup.py test` so I am not really sure :p